app/assets/javascripts/app/contribution/contribution.js

```diff
@@ -16,7 +16,11 @@ App.addChild('Contribution', {
   submitForm: function(event){
     var $target_row = $(event.target).parents('.back-reward-money'),
         user_value = this.$('.selected').find('.user-reward-value').val().replace(/\./g,'');
-    this.$value.val(user_value);
+    if(user_value === ''){
```

Comment:
If we did this there would be no error when the user chooses a smaller value. The user would proceed to the next screen with a different value than he chose, and he might not notice the change.